dd-sdk-android-androidx/src/test/kotlin/com/datadog/android/androidx/tracking/JetpackExtraAttributesProviderTest.kt

```diff
@@ -0,0 +1,151 @@
+package com.datadog.android.androidx.tracking
+
+import android.view.View
+import android.view.ViewGroup
+import android.view.ViewParent
+import android.widget.LinearLayout
+import androidx.recyclerview.widget.RecyclerView
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.whenever
+import fr.xgouchet.elmyr.Forge
+import fr.xgouchet.elmyr.junit5.ForgeExtension
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.api.extension.Extensions
+import org.mockito.Mock
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.junit.jupiter.MockitoSettings
+import org.mockito.quality.Strictness
+
+@Extensions(
+    ExtendWith(ForgeExtension::class),
+    ExtendWith(MockitoExtension::class)
+)
+@MockitoSettings(strictness = Strictness.LENIENT)
+class JetpackExtraAttributesProviderTest {
+
+    @Mock
+    lateinit var mockedRecyclerView: RecyclerView
+
+    @Mock
+    lateinit var mockedTarget: View
+
+    // region Unit tests
+
+    @Test
+    fun `will add the adapter position event if the target is not a RecyclerView direct child`(
```

Comment:
`is not a RecyclerView direct child` is misleading as a view that is not within a RecyclerView's hierarchy is not a RecyclerView's direct child either. 
`if the target is a RecyclerView's nested child` tells that it's both within a RV's hierarchy AND not a direct child.